metricflow/engine/models.py

```diff
@@ -78,12 +79,18 @@ def from_pydantic(cls, pydantic_dimension: SemanticManifestDimension, path_key:
             element_name=path_key.element_name,
             entity_links=path_key.entity_links,
         ).qualified_name
+        parsed_type_params: Optional[DimensionTypeParams] = None
```

Comment:
what's the difference here ?

Reply:
currently, we just pull it from the PydanticDimension which just renders to `TimeGranularity.DAY` (I think that just gets pulled from the config) whereas the `path_key.time_granularity` has the correct rendered granularity.